examples/c48_config.yml

```diff
@@ -0,0 +1,306 @@
+data_table: default
```

Comment:
There are two test configurations already defined in `tests/pytest/config` - should we use the same configurations when testing on HPC?

Reply:
Have removed the c48 configuration and am now pointing to the default c12 configuration in the pytest/config subdirectory